jobs/src/main/scala/dcos/metronome/utils/time/Clock.scala

```diff
@@ -13,6 +13,11 @@ class SystemClock(dateTimeZone: DateTimeZone = DateTimeZone.UTC) extends Clock {
 }
 
 class FixedClock(var _now: DateTime) extends Clock {
+  private[this] var subscribers: List[() => Unit] = Nil
+  def onChange(fn: () => Unit): Unit = synchronized {
+    subscribers = fn :: subscribers
+  }
+
```

Comment:
Where is this subscribers used? My reading of the code indicates that it is unused.

Reply:
This is used in `SimulatedScheduler`

Reply:
How though? How does it get to `SimulatedScheduler`? It's a private[this] var so not accessible via the outside, and, there's only one reference in this class. I know it's possible I'm missing something, because I often do. So, what am I missing? :-)

Reply:
See it :) it's used on line 26 👇